calcom/cal.com
# fix: fetch next batch if event types are empty and cursor exists

## What does this PR do?



- Fixes https://github.com/calcom/cal.com/issues/17107 (GitHub issue number)
- Fixes CAL-4516 (Linear issue number - should be visible at the bottom of the GitHub issue description)

## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?

Login in as enterprise@example.com

1) Go to /event-types 
2) and try creating new user event types,managed event types or team events 

Check whether all event types are loaded
